cryptography/src/bloomfilter.rs

```diff
@@ -20,6 +19,9 @@ const HALF_DIGEST_LEN: usize = 16;
 /// The length of a full [Digest].
 const FULL_DIGEST_LEN: usize = Digest::SIZE;
 
+const BITVEC_CHUNK_SIZE: usize = 1;
+type BitVec = commonware_utils::bitvec::BitVec<BITVEC_CHUNK_SIZE>;
```

Comment:
I think this alias is nicer than doing `BitVec<BITVEC_CHUNK_SIZE>` in a bunch of places. Same in other files.